pycqed/measurement/calibration_points.py

```diff
@@ -185,7 +185,13 @@ def extend_sweep_points(self, sweep_points, qb_name):
         if len(sweep_points) == 0:
             log.warning("No sweep points, returning a range.")
             return np.arange(n_cal_pts)
-        step = np.abs(sweep_points[-1] - sweep_points[-2])
+        try:
+            step = np.abs(sweep_points[-1] - sweep_points[-2])
+        except IndexError:
+            log.warning(f"Less than 2 sweep point detected in: {sweep_points}"
+                        "Could not find appropriate step to extend sweep "
+                        "points. Taking default step of 1")
+            step = 1
```

Comment:
I would not set this to 1. Sometimes our sweep points are pulse lengths in ns or us and it would look weird if the step is 1. I would set the default value to sweep_points[0] for example.  What do you think?